webview/src/experiments/components/table/body/TableContent.tsx

```diff
@@ -52,11 +49,9 @@ export const TableContent: React.FC<TableContentProps> = ({
 
   return (
     <>
-      {branches.map(branch => {
-        const branchRows = rows.filter(row => row.original.branch === branch)
```

Comment:
[F] probably just because the focus is on efficiency this week but I decided that we should get this collection done in a single loop.